### PR TITLE
Used pin instead of attachTo for ContextualBalloon.

### LINKED ISSUE #192: ContextualBalloon should use keepAttachedTo instead of attachTo
Balloon should stick to the target element on scroll and resize.

### PR DESCRIPTION
**Pull request by oskarwrobel**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Used `BalloonPanelView#pin()` instead of `BalloonPanelView#attachTo()` for the `ContextualBalloon` plugin. Closes #192.

---

Related: https://github.com/ckeditor/ckeditor5-link/pull/102

### CODE REVIEW

**Comment by oskarwrobel** [comment]
Out of date after: https://github.com/ckeditor/ckeditor5-ui/pull/197

File: src/contextualballoon.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ export default class ContextualBalloon extends Plugin {
 	 */
 	_show( view ) {
 		this.view.content.add( view );
-		this.view.attachTo( this._getBalloonPosition() );
+		this.view.keepAttachedTo( this._getBalloonPosition() );
```
Comment (oleq) [inline src/contextualballoon.js]: It's `pin()` now https://github.com/ckeditor/ckeditor5-ui/pull/197.

File: tests/contextualballoon.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ import Template from '../src/template';
 import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
 import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
 
-/* global document */
+/* global document, window, Event */
```
Comment (oleq) [inline tests/contextualballoon.js]: Please use `global.window|document`.

File: tests/contextualballoon.js [hunk 3]
```diff
@@ -102,6 +102,23 @@ describe( 'ContextualBalloon', () => {
 			expect( balloon.view.attachTo.firstCall.args[ 0 ] ).to.deep.equal( { target: 'fake' } );
 		} );
 
+		it( 'should update balloon position on scroll and resize', () => {
+			balloon.add( {
+				view: viewA,
+				position: { target: document.createElement( 'div' ) }
+			} );
+
+			sinon.assert.calledOnce( balloon.view.attachTo );
```
Comment (oleq) [inline tests/contextualballoon.js]: I think it's enough to check if `pin()` is called. `ContextualBalloon` doesn't have to verify whether `BalloonPanelView#pin()` does it's job or not, there are `BalloonPanelView` tests to make sure it works.

**Pull request merged**